src/python/metrics/fuzzer_stats_schema.py

```diff
@@ -0,0 +1,426 @@
+# Copyright 2019 Google LLC
```

Comment:
putting the schema here instead of bot/fuzzers/* since App Engine should not import from bot. 

open to suggestions to better places to put this though!

Reply:
no good ideas here. will think more on how to add new engines easily.